Gems/Terrain/Code/Source/Components/TerrainWorldComponent.cpp

```diff
@@ -128,4 +130,42 @@ namespace Terrain
         }
         return false;
     }
-}
+
+    float TerrainWorldConfig::NumberOfSamples(AZ::Vector3* min, AZ::Vector3* max, AZ::Vector2* heightQuery)
+    {
+        float numberOfSamples = ((max->GetX() - min->GetX()) / heightQuery->GetX()) * ((max->GetY() - min->GetY()) / heightQuery->GetY());
+        return numberOfSamples;
```

Comment:
nit: should this check that the arguments aren't nullptr?

Reply:
The values are coming either directly from the RPE or as pointers to the class variables.